openlibrary/components/IdentifiersInput.vue

```diff
@@ -115,40 +114,37 @@ export default {
     },
 
     computed: {
-        popularEditionConfigs: function() {
-            if (this.edition_popular) {
-                const popularConfigs = JSON.parse(decodeURIComponent(this.edition_popular));
-                return Object.fromEntries(popularConfigs.map(e => [e.name, e]));
-            }
-            return {};
+        parsedConfig: function() {
+            return JSON.parse(decodeURIComponent(this.id_config_string))
         },
-        secondaryEditionConfigs: function() {
-            if (this.secondary_identifiers) {
-                const secondConfigs = JSON.parse(decodeURIComponent(this.secondary_identifiers));
-                return Object.fromEntries(secondConfigs.map(e => [e.name, e]));
+        popularIdDict: function() {
```

Comment:
Ok, looking great! This can now just be a list, since we never have to do a look-up by identifier name, which makes this code much tighter!

```
        popularIds: function() {
            const popularIdNames = JSON.parse(decodeURIComponent(this.popular_ids));
            return this.parsedConfig
                .filter((config) => popularConfig.includes(config.name));
        },
```